_guide/_pages/architecture-reviews.md

```diff
@@ -4,13 +4,23 @@ sidenav: approach
 sticky_sidenav: true
 ---
 
-Though we pride ourselves on our [transparent and remote-friendly
-workplace](https://18f.gsa.gov/2015/10/15/best-practices-for-distributed-teams/),
-our project focus tends to silo engineers from each other. We can kick-start
-knowledge transfer by highlighting some of the more interesting design
-decisions from existing projects.
+Maintainable technology projects require handoffs between developers, and with new teammates comes fresh perspectives. Building a [transparent and remote-friendly
+workplace](https://18f.gsa.gov/2015/10/15/best-practices-for-distributed-teams/) is a great start to assist in knowledge transfer, as well as keeping projects as simple and obvious as possible and documenting key decisions. 
 
-## Projects
+## Simplicity
 
-- [DATA Act Pilot: Simplicity is Key]({{site.baseurl}}/architecture-reviews/data-act-pilot) (2016)
-- [Micro-purchase: Do one thing well]({{site.baseurl}}/architecture-reviews/micro-purchase) (2016)
+We’ve done two projects exploring different aspects of simplicity — first, the [DATA Act Pilot: Simplicity is Key]({{site.baseurl}}/architecture-reviews/data-act-pilot) (2016) project explored the ideas of:
+
+- Building for a least common denominator (CSVs) gave the project reach (more users could participate) and reduced code complexity. 
+- Pulling out validation rules into a separate, easy-to-modify format made the product flexible and simple to maintain. 
+
+The second explores the idea of simplifying acquisitons in [Micro-purchase: Do one thing well]({{site.baseurl}}/architecture-reviews/micro-purchase) (2016) by using code boundaries in projects to define lines between micro-purchases of developer time. 
```

Comment:
s/acquisitons/acquisitions

Reply:
Whoops! Fixed in 0100954, thanks for catching!